### PR TITLE
BZ#2086986 remove references to instance_type_id parameter

### PR DESCRIPTION
**Pull request by emarcusRH**
[BZ#2086986](https://bugzilla.redhat.com/show_bug.cgi?id=2086986)

Version(s):
PR applies to enterprise-4.11 or later

Issue:

Remove references to "instance_type_id" parameter from the documentation 
This applies specifically to  

- sample YAML for creating a machineset on RHV, 

and to 
- example installation config YAML example for installing on RHV with customizations

Link to docs preview:
http://file.tlv.redhat.com/emarcus/2086986/installing/installing_rhv/installing-rhv-customizations.html#installation-configuration-parameters-additional-machine_installing-rhv-customizations

http://file.tlv.redhat.com/emarcus/2086986/machine_management/creating_machinesets/creating-machineset-rhv.html#machineset-yaml-rhv_creating-machineset-rhv

Additional information:

### CODE REVIEW

**Comment by emarcusRH** [comment]
@engelmi please review
@Darth-Mera  please review

File: modules/machineset-yaml-rhv.adoc [hunk 1]
```diff
@@ -84,8 +83,6 @@ $ oc get -o jsonpath='{.status.infrastructureName}{"\n"}' infrastructure cluster
 <6> Specify the UUID for the {rh-virtualization} cluster to which this VM instance belongs.
 
 <7> Specify the {rh-virtualization} VM template to use to create the machine.
-
-<8> Optional: Specify the VM instance type.
 +
 [WARNING]
```
Comment (engelmi) [inline modules/machineset-yaml-rhv.adoc:87]: This warning regarding the deprecation of the `instance_type_id` should be removed as well.

**Review by engelmi — CHANGES_REQUESTED** [review]

**Review by engelmi — APPROVED** [review]

**Comment by emarcusRH** [comment]
BZ comment from QE: 
"Veronika Fuxova 2022-07-18 16:46:49 UTC
Hi Eli, 
yes, it's enough to remove just the line "instance_type_id: <instance_type_id> <8>" from this example. In general, remove all the lines that describe instance_type_id, no other lines, please. If you delete all those lines you mention in your first comment, that should be enough."

**Review by jeana-redhat — APPROVED** [review]
LGTM

**Comment by jeana-redhat** [comment]
Picked up in #48399 due to weird commit squash/build errors

**Pull request closed without merging**